src/test/java/com/algorand/algosdk/transaction/TestTransaction.java

```diff
@@ -81,6 +83,7 @@ public void testPaymentTransactionJsonSerialization() throws Exception {
                 .firstValid(301)
                 .lastValid(1300)
                 .genesisHash(new Digest())
+                .fee(0)
```

Comment:
What happens if this line is not included? In order to not have a breaking change, we'll want the behavior for existing code like this to remain the same.

Reply:
I updated the default fee in Transaction to 0 so it's consistent with the default in TransactionBuilder and removed this line.